.github/workflows/build-on-windows.yml

```diff
@@ -170,6 +170,27 @@ jobs:
           name: test-results
           path: test_results_*.xml
 
+      - name: 🔽 Install 'OpenCppCoverage' and 'codecov.io Report Uploader'
```

Comment:
I just read a bit on opencppcoverage. You should only run this on Debug builds, otherwise because otherwise a lot of lines get skipped due to elimination during optimization.

I think we need to add a debug build to the matrix and maybe run code coverage only on the debug build.

Reply:
Ok, added a `Debug` build to the matrix.